doc/manuals/orion-api.md

```diff
@@ -58,6 +58,11 @@
             - [List Entity Types [GET /v2/type]](#list-entity-types-get-v2type)
             - [Retrieve entity information for a given type [GET /v2/types]](#retrieve-entity-information-for-a-given-type-get-v2types)
     - [Subscriptions Operations](#subscriptions-operations)
+        - [Subscription payload datamodel](#Subscriptionpayloaddatamodel)
```

Comment:
```suggestion
        - [Subscription payload datamodel](#subscriptionpayloaddatamodel)
```

Reply:
done in https://github.com/telefonicaid/fiware-orion/pull/4158/commits/88575a0a00f9e89f0fa45e273506a4186183febb